src/main/java/witchinggadgets/WitchingGadgets.java

```diff
@@ -76,6 +77,8 @@ public class WitchingGadgets {
 
     public static SimpleNetworkWrapper packetHandler;
 
+    public static final boolean IS_DREAMCRAFT_LOADED = Loader.isModLoaded("dreamcraft");
```

Comment:
This should probably be fine; but if there's any issue with it move setting it to `preInit`

Reply:
This **is** fine, because the Class is first loaded by FML after all mods have been discovered and the Loader is setup